pom.xml

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <version>1.447</version>
+      <version>1.554.1</version>  <!-- What version does maintainer want here? I added to get @DataBoundSetter -->
```

Comment:
@devennorton yeah just discovered that myself. I should have checked back here and read your comment coz it would have saved me some time :smile: 

Strange one for sure though.